drracket-core-lib/drracket/private/language-object-contract.rkt

```diff
@@ -23,12 +23,12 @@
                                        (let loop ([s #'arg])
                                          (cond
                                            [(syntax? s) 
-                                            (let ([loc (vector (syntax-source s)
-                                                               (syntax-line s)
-                                                               (syntax-column s)
-                                                               (syntax-position s)
-                                                               (syntax-span s))])
-                                              (make-sloc (loop (syntax-e s)) loc))]
+                                            (vector (syntax-source s)
```

Comment:
This doesn't look right to me @jackfirth

Reply:
Hmm. I'm pretty sure this is caused by some issues in Resyntax's ability to track identifier uses at phase 1.